src/common/components/Navbar/Navbar.jsx

```diff
@@ -97,15 +87,15 @@ const Navbar = () => {
   const handleSignIn = async () => {
     dispatch(login());
   };
-  
-  
 
   const handleProfileClick = () => {
-    setIsProfileDropdownOpen(!isProfileDropdownOpen);
+    navigate("/profile");
+    setIsProfileDropdownOpen(false);
   };
 
   const handleSignOut = () => {
     signOut();
+    navigate("/"); // Navigate to home or login page after sign out
```

Comment:
this line is not needed...we configured cognito in such a way that whenever signOut() method from cognito is called it redirects us to home page